src/Life/Main/Init.hs

```diff
@@ -35,8 +56,42 @@ lifeInit owner = do
         copyFile lifeFilePath lifePath
         createRepository owner (Repo "dotfiles")
   where
-    writeConf :: IO ()
-    writeConf = do
-        infoMessage "Writing .life configuration file"
-        let lifeConfig = singleFileConfig lifePath
+    createLifeFile :: IO ()
+    createLifeFile = do
+        infoMessage "Checking existence of some commonly used predefined files..."
+        (exist, noExist) <- scanConfig predefinedLifeConfig
+
+        unless (noExist == mempty) $ do
+            infoMessage "The following files and directories weren't found (so just ignoring them):"
```

Comment:
(so just ignoring them) sounds a bit informal. Probably which will be ignored? Or even better, but I can't think of